webapp/components/survey/CategoryDetails/LevelDetails/LevelDetails.js

```diff
@@ -95,18 +78,7 @@ const LevelDetails = (props) => {
 
       <div className="category__level-items">
         {items.map((item) => (
-          <ItemEdit
-            key={CategoryItem.getUuid(item)}
-            lang={lang}
-            category={category}
-            level={level}
-            item={item}
-            active={CategoryItem.getUuid(item) === activeItemUuid}
-            putCategoryItemProp={putCategoryItemProp}
-            setCategoryItemForEdit={setCategoryItemForEdit}
-            deleteCategoryItem={deleteCategoryItem}
-            readOnly={readOnly}
-          />
+          <ItemDetails key={CategoryItem.getUuid(item)} level={level} item={item} state={state} setState={setState} />
```

Comment:
maybe ItemDetails is a sub component of LevelDetails? is it used only here? if yes, move it under this folder

Reply:
right